cmd/csi-snapshotter/main.go

```diff
@@ -63,6 +60,7 @@ var (
 	snapshotNamePrefix     = flag.String("snapshot-name-prefix", "snapshot", "Prefix to apply to the name of a created snapshot")
 	snapshotNameUUIDLength = flag.Int("snapshot-name-uuid-length", -1, "Length in characters for the generated uuid of a created snapshot. Defaults behavior is to NOT truncate.")
 	showVersion            = flag.Bool("version", false, "Show version.")
+	threads                = flag.Int("threads", 10, "Number of worker threads.")
```

Comment:
Can we use the same flag name as the other sidecars?

Reply:
I've updated this to be `worker-threads` to match the external-attacher.